droid-hal-device-img-boot.inc

```diff
@@ -73,6 +75,12 @@ BuildRequires:  yamui
 BuildRequires:  openssh-clients
 BuildRequires:  openssh-server
 
+%if 0%{?embed_recovery:1}
+BuildRequires: shpp
```

Comment:
Your contribution of shpp is appreciated, but I'm afraid it cannot be justified in the context of HW adaptation recovery, as we are trying to keep dependencies to a minimum. In this particular use-case a `sed` command should be sufficient.